report/index.html

```diff
@@ -51,457 +52,479 @@
       .input-sources>li {
         margin-bottom: 1em;
       }
+
+      .wordmark img {
+        width: 70%;
+      }
     </style>
 
     <title>WebXR Report</title>
   </head>
   <body>
-    <header>
-        <h1>WebXR Report</h1>
+    <div class='container' id='container'>
```

Comment:
Diff is big because I had to indent a lot of stuff but there's only a couple of substantive changes in here, the addition of the header being one of them.